api/search.go

```diff
@@ -117,6 +118,33 @@ func parseURIPrefix(ctx context.Context, params url.Values) (uriPrefix string, e
 	return uriPrefix, nil
 }
 
+// validateCDIDs checks that all the provided CDIDs are not blank and 4 characters long
```

Comment:
```suggestion
// validateCDIDs checks that all the provided CDIDs are not blank and 4 or 5 characters long
```
if the regexp is to be believed

Reply:
I only ever managed to find ones that were 4 or 5 characters long, 'ukpop' being 5. Do we have any standards when it comes to cdids?

Reply:
was just interested in the "4 or 5" element of it :D